### ISSUE TITLE
Background Image doesnt appear in getData

### ISSUE THREAD

**Issue by Binternet** [willowsystems/jSignature#104]
I've managed to import a background image to the canvas using `setData` method
Yet when I export using `getData` for `svgbase64` - only the strokes appear and the background image is gone.

Any ideas?

**Comment by coriam** [comment]
Did you ever figure this out?  I was going to attempt something similar...

**Comment by Binternet** [comment]
@coriam yeah, I switched to [Signature Pad](https://github.com/szimek/signature_pad) 🙊
